DashAI/back/api/api_v1/endpoints/plugins.py

```diff
@@ -240,6 +251,9 @@ async def update_plugin(
     with session_factory() as db:
         try:
             plugin = db.get(Plugin, plugin_id)
+            plugin_name = plugin.name
+            install_plugin_from_pypi(plugin_name)
+            register_new_plugins(component_registry)
```

Comment:
Related with the comment that I left before, here I think that you should execute `get_available_plugins`

Reply:
Perfect, its ready